test/parallel/test-stdin-from-file.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 const common = require('../common');
+const { path } = require('../common/fixtures');
```

Comment:
This is somewhat confusing because it has the same name as the `path` module and therefore I would normally expect it to be the path module. It would be nicer to use the fixture name here instead.